middleware/job/api.py

```diff
@@ -1,7 +1,20 @@
 import json_merge_patch
 from flask_restful import Resource, abort, request
-from middleware.job.model import is_valid_job_json, job_summary_json
 from middleware.job_information_manager import job_information_manager as JIM
+from middleware.job.schema import job_to_json, json_to_job, JobSchema
+
+
+def is_valid_job_json(job):
```

Comment:
Is it not clearer to have these functions and any other helper fns reside in a separate file rather than polluting the namespace of `api.py`?